chore: debugging (#96)
zane-ops/zane-ops: CONTRIBUTING.md

   The app should be available at https://app.zaneops.local.

> 7. **Debugging :**

Reviewer: This should be its own section
```suggestion
##  🛠️ Debugging 
```

   The app should be available at https://app.zaneops.local.

> 7. **Debugging :**

Reviewer: This should be its own section
```suggestion
##  🛠️ Debugging 
```

>    If the app is unresponsive

Reviewer: ```suggestion
   You may end up having issues where the project is not working, the app is not reachable on the browser, or the API seems to be down, this section is to help debugging this case, if the app is working fine on your end, you don't need to read this section.
```

We need a better leading paragraph here to explain what this section is.

>    0. make sure you ran make dev and it didn't exit unexpectedly 

Reviewer: ```suggestion
   1. make sure you ran `make dev` and it didn't exit unexpectedly 
```

>    1. make sure that all the containers are up, you can check it in your docker tool of choice, orbstack or docker desktop 

Reviewer: ```suggestion
   2. make sure that all the containers are up, you can check it in your docker tool of choice, orbstack or docker desktop 
```

   ![Alt text](https://media.discordapp.net/attachments/1146204833913782363/1234480044492198010/image.png?ex=6630e2a2&is=662f9122&hm=51504fb845b7e5f5cd36ea50908ceced025b4bbb60e9b7cf177cc53a8adff745&=&format=webp&quality=lossless&width=1880&height=1202)

>    2. make sure that the API is launched, and that no error is in thrown in the terminal where make dev is running

Reviewer: ```suggestion
   3. make sure that the API is launched, and that no error is in thrown in the terminal where `make dev` is running
```

>    3. make sure to run nslookup

Reviewer: ```suggestion
   4. make sure to run `nslookup` as stated in the previous section detailling how to setup the local domain
```

>    4. make sure to setup the project and install the packges with make setup

Reviewer: ```suggestion
   5. make sure to setup the project and install the packages with `make setup`
```

>    5. If the app is still unresponsive, run make reset-db

Reviewer: ```suggestion
   6. If the app is still unresponsive, run `make reset-db`
```

Also add a warning to say that this command will reset the project from scratch and delete everything
